scp/lib/ui/cards.dart

```diff
@@ -176,6 +178,148 @@ Widget appointmentCard(BuildContext context) {
     ),
   );
 }
+Widget yourDostCard(BuildContext context) {
+  Gradients().init(context);
+  SizeConfig().init(context);
+  double heightFactor = SizeConfig.screenWidth;
+  return SizedBox(
+    height: heightFactor * 0.58,
+    child: Padding(
+      padding: const EdgeInsets.only(top: 12.0),
+      child: InkWell(
+        onTap: () async {
+          launchLink("https://yourdost.com/");
+
+        },
+        // onTap: () async {
+        //   SharedPreferences prefs = await SharedPreferences.getInstance();
+        //   if (prefs.getBool('hasBooked') == true) {
+        //     Navigator.of(context).push(MaterialPageRoute(
+        //         builder: (BuildContext context) => Booking(
+        //             /*keyCode: gKey,*/ counselDay: gCounselDay, time: gTime)));
+        //   } else
+        //     Navigator.of(context).pushNamed('/appointments');
+        // },
```

Comment:
Why are these comments needed here?

Reply:
The code will be shifted and archived at some other place so that it can be used in future if the appointments will be booked for offline counseeling

Reply:
So why is that comment here? Shouldn't that be there in the existing appointment card?

Reply:
Yeah, it should be. I've pointed it out to the developer. It'll be taken care.